src/components/common/ColonyMembers/MemberControls/MemberControls.tsx

```diff
@@ -0,0 +1,186 @@
+  editPermissions: {
+    id: `${displayName}.editPermissions`,
+    defaultMessage: 'Edit permissions',
+  },
+  banAddress: {
+    id: `${displayName}.banAddress`,
+    defaultMessage: 'Ban address',
+  },
+  unbanAddress: {
+    id: `${displayName}.unbanAddress`,
+    defaultMessage: 'Unban address',
+  },
+  manageWhitelist: {
+    id: `${displayName}.manageWhitelist`,
+    defaultMessage: 'Manage address book',
+  },
+});
+
+type Props = {
+  isRootDomain: boolean;
+};
+
+const MemberControls = ({ isRootDomain }: Props) => {
+  const { colony } = useColonyContext();
+  // const {
+  //   networkId,
+  //   username,
+  //   ethereal,
+  //   walletAddress: currentUserWalletAddress,
+  // } = useLoggedInUser();
+  // const {
+  //   user: { name },
+  //   wallet: currentUserWalletAddress,
+  // } = useAppContext();
+
+  // const openToggleBanningDialog = useDialog(BanUserDialog);
+
+  // const { data: colonyExtensions } = useColonyExtensionsQuery({
+  //   variables: { address: colonyAddress },
+  // });
+
+  // const openPermissionManagementDialog =  useDialog(PermissionManagementDialog);
+
+  const handlePermissionManagementDialog = useCallback(() => {
+    // eslint-disable-next-line no-console
+    console.log('Open permissions dialog');
+    // openPermissionManagementDialog({
+    //   colony,
+    // });
+  }, []);
+
+  // const openToggleManageWhitelistDialog = useDialog(ManageWhitelistDialog);
+
+  const handleToggleWhitelistDialog = useCallback(() => {
+    // eslint-disable-next-line no-console
+    console.log('Open whitelist dialog');
+    // return openToggleManageWhitelistDialog({
+    //   colony,
+    // });
+  }, []);
+
+  // eslint-disable-next-line max-len
+  // const oneTxPaymentExtension =
+  //   colonyExtensions?.processedColony?.installedExtensions.find(
+  //     ({ details, extensionId: extensionName }) =>
+  //       details?.initialized &&
+  //       !details?.missingPermissions.length &&
+  //       extensionName === Extension.OneTxPayment,
+  //   );
+  // const mustUpgradeOneTx = oneTxMustBeUpgraded(oneTxPaymentExtension);
+
+  // const isSupportedColonyVersion =
+  //   parseInt(version || '1', 10) >= ColonyVersion.LightweightSpaceship;
+
+  // const currentUserRoles = useTransformer(getAllUserRoles, [
+  //   colony,
+  //   currentUserWalletAddress,
+  // ]);
+  const canMamangePermissions = true;
+  //   (hasRegisteredProfile && canArchitect(currentUserRoles)) ||
+  //   hasRoot(currentUserRoles);
+  const canAdministerComments = true;
+  //   hasRegisteredProfile &&
+  //   (hasRoot(currentUserRoles) || canAdminister(currentUserRoles));
+  const canManageWhitelist = true;
+  //  hasRegisteredProfile && hasRoot(currentUserRoles);
+
+  const controlsDisabled = false;
+  // !isSupportedColonyVersion ||
+  // !isNetworkAllowed ||
+  // !hasRegisteredProfile ||
+  // !isDeploymentFinished ||
+  // mustUpgradeOneTx;
+
+  return (
+    !controlsDisabled && (
+      <ul className={styles.controls}>
```

Comment:
@willm30 I think this can wait for now. I removed the ban/unban and whitelist functionality so there is only one option, the invitelink button, at the moment.